### PR TITLE
Dbfix2

### PR DESCRIPTION
**Pull request by SvenHiemstra**

### CODE REVIEW

File: WpfApp1/project_1_game_inteact/project_1_game_inteact/resources/datastorage.json
Comment (bestman8) [inline WpfApp1/project_1_game_inteact/project_1_game_inteact/resources/datastorage.json:1]: waarom 
    "**/bin",
    "**/bower_components",
    "**/jspm_packages",
    "**/node_modules",
    "**/obj",
    "**/platforms"

File: WpfApp1/project_1_game_inteact/project_1_game_inteact/start.xaml.cs [hunk 1]
```diff
@@ -56,7 +56,7 @@ private void levels_Click(object sender, RoutedEventArgs e)
 
         private void handleiding2_Click(object sender, RoutedEventArgs e)
         {
-            project_1_game_inteact.handleiding handleiding2 = new handleiding();
+            project_1_game_inteact.Leaderboard handleiding2 = new Leaderboard();
```
Comment (bestman8) [inline WpfApp1/project_1_game_inteact/project_1_game_inteact/start.xaml.cs:59]: uhm

File: WpfApp1/project_1_game_inteact/project_1_game_inteact/endscreen.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+            project_1_game_inteact.Leaderboard lead = new project_1_game_inteact.Leaderboard();
+            lead.Show();
+            this.Close();
+        }
+
+        private void levels_Click(object sender, RoutedEventArgs e)
+        {
+            UItest.levels levels = new UItest.levels();
+            levels.Show();
+        }
```
Comment (bestman8) [inline WpfApp1/project_1_game_inteact/project_1_game_inteact/endscreen.xaml.cs:114]: this.close

File: WpfApp1/project_1_game_inteact/project_1_game_inteact/resources/datastorage.json
Comment (SvenHiemstra) [reply WpfApp1/project_1_game_inteact/project_1_game_inteact/resources/datastorage.json:1]: Dat is standaard

**Review by bestman8 — APPROVED** [review]

**Pull request merged**